pkg/object/objmetadata_set.go

```diff
@@ -130,34 +130,27 @@ func (setA ObjMetadataSet) Diff(setB ObjMetadataSet) ObjMetadataSet {
 	return diff
 }
 
+// Unique returns the set with duplicates removed.
+// Order may or may not remain consistent.
+func (setA ObjMetadataSet) Unique() ObjMetadataSet {
+	return ObjMetadataSetFromMap(setA.ToMap())
+}
+
 // Hash the objects in the set by serializing, sorting, concatonating, and
 // hashing the result with the 32-bit FNV-1a algorithm.
-func (setA ObjMetadataSet) Hash() (string, error) {
+// Used to quickly identify the set of resources in the inventory object.
+func (setA ObjMetadataSet) Hash() string {
```

Comment:
This hash change can probably be extracted to a different PR. It's needed for this PR, but can be made standalone.